apps/web-giddh/src/app/theme/account-detail-modal/account-detail-modal.component.ts

```
@@ -129,7 +129,7 @@ export class AccountDetailModalComponent implements OnChanges, OnDestroy {
                 break;
 
             case 1: // go to ledger
-                const encodedAccountUniqueName = this.generalService.getEncodedValue(this.accountUniqueName);
+                const encodedAccountUniqueName = this.generalService.getBase64EncodedString(this.accountUniqueName);
```

Biome (1.9.4)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)